projects/storefrontlib/src/cms-components/navigation/footer-navigation/footer-navigation.component.ts

```diff
@@ -30,7 +32,8 @@ export class FooterNavigationComponent {
     protected componentData: CmsComponentData<CmsNavigationComponent>,
     protected service: NavigationService,
     protected config: AnonymousConsentsConfig,
-    protected authService: AuthService
+    protected authService: AuthService,
```

Comment:
Breaking change mate, you got to do the whole constructor deprecation thing.

Reply:
This ticket takes care of breaking changes and adds a feature level flag - https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/4394